editor/editor_help.h

```diff
@@ -35,6 +35,7 @@
 #include "editor/code_editor.h"
 #include "editor/doc_tools.h"
 #include "editor/editor_plugin.h"
+#include "modules/modules_enabled.gen.h"
```

Comment:
The convention is to include this after the main block of includes, with a comment explaining why it's used:
```
...
#include "scene/gui/split_container.h"
#include "scene/gui/tab_container.h"
#include "scene/gui/text_edit.h"
#include "scene/main/timer.h"

#include "modules/modules_enabled.gen.h" // For gdscript, mono.
```

And while it's redundant, we include it in each file that uses a `MODULE_.*_ENABLED`, so this should be in the `.cpp` too, even if it's available through the `.h`. Otherwise it's risky to eventually refactor the `.h` to remove this dependency, while it's still needed in the `.cpp` and fails silently.